lib/zstd.h

```diff
@@ -1796,6 +1798,13 @@ ZSTDLIB_API size_t ZSTD_CCtx_refPrefix_advanced(ZSTD_CCtx* cctx, const void* pre
  */
 #define ZSTD_c_validateSequences ZSTD_c_experimentalParam12
 
+/* ZSTD_c_splitBlocks
+ * Default is 0 == disabled. Set to 1 to enable block splitting.
```

Comment:
Just FYI :
depending on the way we will want to integrate this capability into normal compression levels,
we may need an additional "auto" value (which would likely use the `0` slot)
which would be interpreted as "enabled" or "disabled" by `zstd`, 
depending on compression level (or more likely based on compression strategy).

There is no urgency to decide that now.